### DISCUSSION TITLE
Wrong decision in one thought!

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#81] [General]
Thorium browser, I feel that it has deviated from the concept of thorium now; although it is said that the number of teams is small and due to stability considerations, it will no longer follow up the upstream version, but even the official version you are slow to follow up, and the original intention is no longer ! 111.0 instead of 112.0, no matter how optimized your functions are, the core follow-up is not timely; followed by a discount for safe browsing experience! .

**Comment by gz83** [comment]
> Thorium browser, I feel that it has deviated from the concept of thorium now; although it is said that the number of teams is small and due to stability considerations, it will no longer follow up the upstream version, but even the official version you are slow to follow up, and the original intention is no longer ! 111.0 instead of 112.0, no matter how optimized your functions are, the core follow-up is not timely; followed by a discount for safe browsing experience! .

@whxuezhe 

Why are you so eager to push for updates? Is there something seriously wrong with the current version?

Besides, each software has its own development plan and the developers will control all aspects. Also, it's not sensible to blindly pursue new versions.

This discussion will be closed soon and there is no point in continuing it.
